plugins/UsersManager/lang/en.json

```diff
@@ -8,7 +8,7 @@
         "AnonymousUserHasViewAccess2": "Your analytics reports and your visitors information are publicly viewable.",
         "ApplyToAllWebsites": "Apply to all websites",
         "ChangeAllConfirm": "Are you sure you want to give '%s' access to all the websites?",
-        "ChangePasswordConfirm": "Changing the password will also change the user's token_auth. Do you really want to continue?",
+        "ChangePasswordConfirm": "Changing the password will also change the user's old token_auth. This will only happen once. Do you really want to continue?",
```

Comment:
I just noticed again this very same message is also displayed when changing password on the personal settings page. Would it be viable to reword it there a bit? Saying "the users" when addressing the user actually affected seems somewhat off... Perhaps just replacing the addressing part using `%s` and siwtching to "your" on that page.